code/presentation/mb-client/src/screens/home/index.tsx

```diff
@@ -1,20 +1,16 @@
+import { useNavigation } from '@react-navigation/native';
 import { FlashList } from '@shopify/flash-list';
 import React from 'react';
+import { TouchableOpacity } from 'react-native-gesture-handler';
 
 import type { MasjidWithDistance } from '@/api/masjid';
 import { useMasjids } from '@/api/masjid/use-masjids';
-import {
-  EmptyList,
-  FocusAwareStatusBar,
-  Pressable,
-  Search,
-  Text,
-  View,
-} from '@/ui';
+import { EmptyList, FocusAwareStatusBar, Text, View } from '@/ui';
 
 import { MasjidCard } from './masjid';
 
 export const HomeScreen = () => {
+  const navigation = useNavigation(); // Correct usage of useNavigation hook
```

Comment:
as we are just using navigate function, we can do ` const { navigate } = useNavigation();` to be consistent with the existing code base

Reply:
Done